packages/flutter_tools/lib/src/application_package.dart

```diff
@@ -28,6 +28,8 @@ abstract class ApplicationPackage {
 
   String get displayName => name;
 
+  String get packagePath => null;
```

Comment:
The class is abstract - do we need to provide an implementation here (`=> null`) - do all subclasses override this?